react/components/GalleryItem.tsx

```diff
@@ -17,6 +17,8 @@ interface GalleryItemProps {
   summary: any
   /** Item position in the gallery */
   position: number
+  /** Name of the list of products */
```

Comment:
Might be useful to expand this just a little to include the purpose of this name. Like "This name will be used when sending pixel events" for example